### PR TITLE
move files around for tutorials reorg

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR reorganizes the tutorial section by product and updates the following:

- [x]  Existing index pages
- [x]  New index pages were created for the new sections
- [x] Addition of the [multichain assets PR](https://github.com/wormhole-foundation/wormhole-docs/pull/148)
- [x] Existing links, snippets, etc.

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [x] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

Redirects PR: https://github.com/papermoonio/wormhole-mkdocs/pull/105

### CODE REVIEW

**Comment by ilariae** [comment]
pr for the icons in mkdocs https://github.com/papermoonio/wormhole-mkdocs/pull/111

File: tutorials/by-product/connect/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<div class="grid cards" markdown>
+
+-   :octicons-tools-16:{ .lg .middle } **Connect**
+
+    ---
+
+    Get deeper insights into setting up and customizing Wormhole Connect. Explore advanced guides, best practices, and configuration tips to streamline your cross-chain integrations.
+
+    [:octicons-arrow-right-16: Start building](/docs/build/applications/connect/)
```
Comment (eshaben) [inline tutorials/by-product/connect/index.md:32]:
```suggestion
    [:octicons-arrow-right-16: Learn more](/docs/build/applications/connect/)
```

File: tutorials/by-product/cross-chain-contracts/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    [:octicons-arrow-right-16: Explore Core Contracts](/docs/learn/infrastructure/core-contracts/)
+
+-   :octicons-tools-16:{ .lg .middle } **Build Contract Integrations**
+
+    ---
+
+    Explore advanced contract integrations with Wormhole. Access reference code, best practices, and guidance for deploying robust cross-chain smart contracts.
+
+
+    [:octicons-arrow-right-16: Build Contract Integrations](/docs/build/contract-integrations/)
```
Comment (eshaben) [inline tutorials/by-product/cross-chain-contracts/index.md:51]:
```suggestion
    [:octicons-arrow-right-16: Build contract integrations](/docs/build/contract-integrations/)
```

File: tutorials/by-product/multigov/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+<div class="grid cards" markdown>
+
+-   :octicons-book-16:{ .lg .middle } **Governance Fundamentals**
+
+    ---
+
+    Dive into Wormhole’s governance mechanisms. Understand how cross-chain governance works, proposal creation, voting, and execution.
+
+    [:octicons-arrow-right-16: Explore Governance](/docs/learn/governance/)
```
Comment (eshaben) [inline tutorials/by-product/multigov/index.md:34]:
```suggestion
    [:octicons-arrow-right-16: Explore governance](/docs/learn/governance/)
```

File: tutorials/by-product/multigov/index.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+-   :octicons-tools-16:{ .lg .middle } **Implement MultiGov**
+
+    ---
+
+    Integrate MultiGov into your smart contracts. Access reference code, best practices, and guidance for deploying cross-chain governance systems.
+
+
+    [:octicons-arrow-right-16: Build MultiGov](/docs/build/contract-integrations/multigov/)
```
Comment (eshaben) [inline tutorials/by-product/multigov/index.md:43]:
```suggestion
    [:octicons-arrow-right-16: Build with MultiGov](/docs/build/contract-integrations/multigov/)
```

File: tutorials/by-product/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+title: Tutorials by Product
+description: Discover product-specific Wormhole tutorials. Learn setup, integration, and advanced features to develop cross-chain apps confidently.
+template: root-index-page.html
```
Comment (eshaben) [inline tutorials/by-product/index.md:4]:
```suggestion
```

File: tutorials/by-product/cross-chain-contracts/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+---
+title: Cross-Chain Contract Tutorials
+description: Master cross-chain smart contracts with Wormhole. Learn messaging, token transfers, and secure, scalable dApp deployments across blockchains.
+template: root-index-page.html
```
Comment (eshaben) [inline tutorials/by-product/cross-chain-contracts/index.md:4]:
```suggestion
```

**Review by ilariae — APPROVED** [review]
LGTM 🚀

**Pull request merged**